### PR TITLE
RHDEVDOCS-2623: Improve the steps for not having pending operations to…

### PR DESCRIPTION
**Pull request by rolfedh**
… be written to disk in ES rollout

- Aligned team: Dev Tools
- For branches:  4.6+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-2623
- Direct link to doc preview: https://deploy-preview-35399--osdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-log-store?utm_source=github&utm_campaign=bot_dp#cluster-logging-manual-rollout-rolling_cluster-logging-store
- Reporter review: @r2d2rnd 
- SME review: @periklis 
- QE review: @kabirbhartiRH 
- Peer review: @jc-berger 
- All reviews complete. Please merge now.

### CODE REVIEW

File: modules/cluster-logging-manual-rollout-rolling.adoc [hunk 1]
```diff
@@ -208,3 +215,10 @@ $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_uti
   }
 }
 ----
+
+. Scale down the fluentd pods so they do not send new logs to Elasticsearch.
```
Comment (rolfedh) [inline modules/cluster-logging-manual-rollout-rolling.adoc:219]: @r2d2rnd Is this really what you meant to say?

**Comment by r2d2rnd** [comment]
One errata is present, it was on my side when creating the bug. Line 9 is now:

~~~
9. Scale down the fluentd pods so they do not send new logs to Elasticsearch.
~~~

When it should be to scale up (not down) to enable again fluentd pods start and send to Elasticsearch

~~~
9. Scale up the fluentd pods so they do not send new logs to Elasticsearch.
~~~

**Comment by rolfedh** [comment]
Hi, @periklis would you review and comment on the updates in this PR in light of the Release Notes Text in this Jira: https://issues.redhat.com/browse/LOG-1536 ? 

Here's a link to the preview: https://deploy-preview-35399--osdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-log-store?utm_source=github&utm_campaign=bot_dp#cluster-logging-manual-rollout-rolling_cluster-logging-store

I'd appreciate if you would also have a look around for other potential doc impacts.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by periklis — COMMENTED** [review]
LGTM

**Pull request merged**